GiEnJul/ClientApp/src/components/admin/editLandingPage/EditCompany.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Editor } from "react-draft-wysiwyg";
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+import { useState } from 'react';
+import { convertToRaw, EditorState } from 'draft-js';
+import draftToHtml from 'draftjs-to-html';
+import { Typography, FormGroup, TextField } from '@material-ui/core';
+
+type Props = {
+    step: Number,
+    companyState: EditorState
+    setCompanyState: (input: EditorState) => void,
+    companyTitleState: string,
+    setCompanyTitleState: (input: string) => void,
+}
+
+
+const EditCompany: React.FC<Props> = ({ step, companyState, setCompanyState, companyTitleState, setCompanyTitleState }) => {
```

Comment:
The step variable can probably be removed now that it's not used in the component anymore. This applies to all these "subpage" components.